tools/validate_spec.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env -S python3 -O
```

Comment:
It is true that the output here is not particularly pretty and nice to read. The options that come to my mind are removing the error message completely from the console output or truncating it to a fixed length or only printing the error class. 
When implementing the message parser of PyRFLX, we had a debate over the error message for opaque fields and removed the shorter printing of the field value for opaque fields in hex in error messages in favour of the pure bit-representation, because the conversion from the bit-representation to hex was the sole line which could not be tested in the typevalue module.

Reply:
> The options that come to my mind are removing the error message completely from the console output or truncating it to a fixed length or only printing the error class.

I think error messages can be removed completely for correctly classified messages. One option would be just printing the test file name and the result (PASSED vs. FAILED), and only provide more information for failed tests (similar to pytest). I wouldn't mind long PyRFLX error messages for failed tests.

Reply:
In 042d9ef the console output has been changed. For passed tests, the file name and PASSED is printed and for failed tests FAILED plus additional information is printed.